pkg/test/integration/server.go

```diff
@@ -33,8 +34,10 @@ import (
 )
 
 const (
-	syncPeriod = "30s"
-	errCleanup = "failure in default cleanup"
+	syncPeriod   = "30s"
+	errCleanup   = "failure in default cleanup"
+	errGetRemote = "unable to download CRDs from remote location"
+	remotePath   = "./tmp-test"
```

Comment:
Is this a temporary directory used to store downloaded remote CRDs?

Would it make sense to use https://golang.org/pkg/io/ioutil/#TempDir rather than a fixed directory? If not, could you add a comment explaining what this directory is for.

Reply:
You are correct, and I like the idea! Pending decisions on comments below, I will implement this using `TempDir` instead :)